docs/faq/differences-with.../meltano-vs-airbyte.md

```diff
@@ -0,0 +1,23 @@
+# Meltano vs Airbyte
+
+### **Meltano:**
+
+Meltano is a Gitlab side project. Since 2019, they have been iterating on several approaches. The latest positioning is an orchestrator dedicated to data integration that was built by Gitlab on top of Singer’s taps and targets. They now have only one maintainer for this project. 
```

Comment:
```suggestion
Meltano is a Gitlab side project. Since 2019, they have been iterating on several approaches. The latest positioning is an orchestrator dedicated to data integration that was built by Gitlab on top of Singer’s taps and targets. They currently have only one maintainer for this project. 
```